src/minigames/minigame.ts

```diff
@@ -7,6 +7,14 @@ export interface MinigameDelegate {
 /** Base class for all minigames to extend. */
 export class Minigame {
   protected container: Container;
+  /**
+   * How long this minigame should go for before timing out
+   * and failing the player. In milliseconds. Set to
+   * undefined for a minigame with no time limit.
+   */
+  protected lifetime = 10_000;
+  protected elapsedMS = 0;
```

Comment:
nit: since this means something a bit different to the ticker.elapsedMS, could we call this cumulativeMS?